src/components/Navbar.tsx

```diff
@@ -0,0 +1,26 @@
+import { useNavigate } from "react-router-dom";
+
+const Navbar = () => {
+    const navigate = useNavigate();
+    
+    // For external URLs, use window.location.href or an anchor tag
+    const handleLogoClick = () => {
+        window.location.href = "https://worldwidewordministries.org/";
```

Comment:
this will cause the page to reload 
which means all data will be refetched this will cause delay in a slow network situation
we could use the router.navigate("/") instead